conf/messages.en

```diff
@@ -465,26 +465,58 @@ pdf.ruling.aboutDecision.text=I have made this decision under section 24 of The
+pdf.ruling.section.askForReview.solsTeam.emailText=The SOLS team email address is: <a class="govuk-link" href="mailto:{0}">{1}</a> If you decide to use email to correspond, please see the important information at the end of this letter. If you want to contact the team in writing the address is –
+pdf.application.footer.solicitors.office.and.legal.services.address=Solicitors Office and Legal Services<br> HM Revenue and Customs<br> BX9 1ZT
+
+pdf.ruling.section.appealForIndependentTribunal.title=If you want to appeal to an independent tribunal
+pdf.ruling.section.appealForIndependentTribunal.appealImmediately=If you would prefer, you can appeal immediately to an independent tribunal. You can also progress to a tribunal appeal once you have received the review outcome and disagree with the decision.
+pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals=Your request must reach HM Courts and Tribunals service within 30 days of the date of this letter, or within 30 days of the date of the letter that tells you the outcome of the review.
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements=The request should include copies of:
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1=this letter, or review outcome letter
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2=any assessment
+pdf.ruling.section.appealForIndependentTribunal.relevantInfo=If you do not send the relevant information, HM Courts and Tribunals may reject your appeal.
+pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo=For more information about tribunals go to the page below or call 0300 123 1024.
+pdf.ruling.section.appealForIndependentTribunal.page=<a href="www.gov.uk/tax-tribunal" target="_blank">www.gov.uk/tax-tribunal</a>
+
+pdf.ruling.section.appealsAndReviews.title=More information about appeals and reviews
+pdf.ruling.section.appealsAndReviews.moreInfo=For more information about your appeal and review rights, go to the following page:
+pdf.ruling.section.appealsAndReviews.page=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision</a>
+pdf.ruling.section.appealsAndReviews.copy=You can ask for a copy of the HMRC1 document by calling 0300 200 3610.
+
+pdf.ruling.section.communicatingByEmail.title=Important information about communicating by email
+pdf.ruling.section.communicatingByEmail.tellUs=If you want to correspond by email, you must first tell us that you understand and accept the risks of using email. These are detailed in our factsheet CC/FS72 DSC1, 'Corresponding with HMRC by email'.  To find this factsheet, go to this page:
+pdf.ruling.section.communicatingByEmail.page=<a href="www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72" target="_blank">www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72</a>
```

Comment:
Please add `class="govuk-link"` within the anchor tag for this link, thanks

Reply:
The href is wrong and redirecting to a not found page. Please use the full url and update the href to `https://www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72` thanks